From 24c7384addb87bb1c1e3e3e5e795c22cabf2532b Mon Sep 17 00:00:00 2001
From: Aman Bhadouria <aman.bhadouria@intel.com>
Date: Wed, 21 Aug 2024 21:20:00 +0530
Subject: [PATCH] Enabling Software Gatekeeper to AIDL

Software Gatekeeper AIDL service should work after
setting Target API and Level that are required for
Android 15

Tests Done:
1. Set the Target API and Level
2. Flash the image Bare metal or CiV
3. Image is booting and service is running

Tracked-On: OAM-123432
Signed-off-by: Aman Bhadouria <aman.bhadouria@intel.com>
---
 groups/tee/false/product.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/tee/false/product.mk b/groups/tee/false/product.mk
index 6dac8ec..33630cc 100644
--- a/groups/tee/false/product.mk
+++ b/groups/tee/false/product.mk
@@ -1,6 +1,6 @@
 PRODUCT_PACKAGES += \
+    com.android.hardware.gatekeeper.nonsecure \
     android.hardware.security.keymint-service
-    #com.android.hardware.gatekeeper.nonsecure \
 
 #PRODUCT_PACKAGES += \
 #    android.hardware.keymaster@3.0-service \
-- 
2.45.2

